citesphere/src/main/webapp/WEB-INF/views/auth/group/editItem.jsp

```diff
@@ -119,6 +178,50 @@ $(function() {
 		resetAuthorCreationModal();
 	});
 	
+	$("#addEditorButton").click(function() {
+		var firstname = $("#firstNameEditor").val();
+		var lastname = $("#lastNameEditor").val();
+		var uri = $("#uriEditor").val();
+		var localAuthority = $("#uriEditorLocalId").val();
+		
+		var editorSpan = $("<span>");
+		editorSpan.attr("class", "label label-info editor-item");
+		editorSpan.attr("data-editor-firstname", firstname);
+		editorSpan.attr("data-editor-lastname", lastname);
+		editorSpan.attr("data-editor-uri", uri);
+		editorSpan.attr("data-editor-authority-id", localAuthority);
+		
+		var affiliationsList = [];
+		var affSpan = $("<span>");
+		$("#editorAffiliations").children().each(function(idx, elem) {
+			var affSpan = $("<span>");
+			var input = $(elem).find("input");
+			affSpan.attr("data-affiliation-name", input.val());
+			affiliationsList.push(input.val());
+			editorSpan.append(affSpan);
+		});
+		
+		var affiliationString = "";
+		if (affiliationsList) {
+			affiliationString = " (" + affiliationsList.join(", ") + ")";
+		}
+		
+		editorSpan.append(lastname + ', ' + firstname + affiliationString + '&nbsp;&nbsp; ');
+		var deleteIcon = $('<i class="fas fa-times remove-editor"></i>');
+		deleteIcon.click(removeEditor);
+		editorSpan.append(deleteIcon);
+		$("#editorList").append(editorSpan);
+		$("#editorList").append("&nbsp;&nbsp; ")
+		
+		$("#editorModal").modal('hide');
```

Comment:
this is more or less the same code as for authors, right? Just put it in a method taking the things that are different as parameters.